### ISSUE TITLE
Request body larger than maxBodyLength Limit

### ISSUE THREAD

**Issue by RacknerFrank** [ConvertAPI/convertapi-node#36]
I have been getting an error trying to split a large pdf with this library.

Here's my code:

```
convertapi.convert('split', {
  File: fileName
}, 'pdf')
```

Here's the error:
```
{ Error: Request body larger than maxBodyLength limit
    at Function.handleError (/node_modules/convertapi/lib/client.js:1:2396)
    at then.catch.a (/node_modules/convertapi/lib/client.js:1:2231)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:229:7)
```

I upgraded to version 1.9.1 and still experience this error

**Comment by tomasr78** [comment]
What is the file size?

**Comment by RacknerFrank** [comment]
11MB

**Comment by tomasr78** [comment]
@laurynas-baltsoft Is there any request body limitation in NodeJS? Can we increase it?

**Comment by laurynas-baltsoft** [comment]
Yes, there was default body limitation in axios library. [Fixed it](https://github.com/ConvertAPI/convertapi-node/commit/ac9930b46d5681c5472296846af323f17c90d1cc) and released `1.10.1` client version.
